docs/index.rst

```diff
@@ -17,7 +17,7 @@ might still make breaking changes that have no clear upgrade pathway.
 Installation
 ============
 
-The Littlest JupyterHub (TLJH) can run on any server that is running **Ubuntu 18.04** or **Ubuntu 20.04** on a amd64 or arm64 CPU architecture. Earlier versions of Ubuntu are not supported.
+The Littlest JupyterHub (TLJH) can run on any server that is running **Debian 10** or **Ubuntu 20.04** on a amd64 or arm64 CPU architecture. Earlier versions of Ubuntu and Debian are not supported, as are other Linux distributions.
```

Comment:
We test with Ubuntu 22.04 as well, so that makes sense to include still, and I figure we look to support Debian 11 as well.

Reply:
Yup, sometimes it says `>=`, but here it implies exact. What do we want to claim? Do we want to claim only exactly `20.04, 22.04`, or `>=20.04`?

Reply:
I figure that we aim to support LTS releases of Ubuntu 20.04 and Debian 11 or higher, and currently we test for Ubuntu 20.04, 22.04, and debian 11.

Reply:
added:

> We aim to support 'stable' and Long-Term Support (LTS) versions.
Newer versions are likely to work with little or no adjustment, but these are the officially supported and tested versions.

to try to communicate that 24.04 _may_ work, but if it's not in the list above, it's not 'supported'

Reply:
Looked at the commit, I think it was great!!